### ISSUE TITLE
Unknown script "start". 

### ISSUE THREAD

**Issue by PiPauwels** [kat-tax/ult#20]
I feel like a total idiot for having to ask this but I did these exact steps (as documented):

`npx ult UltTest`
(squeezed a `cd UltTest` in between here)
`npm run web`

Result: `Unknown script "start". Perhaps you need to update ULT?`

Tried again after an `npm i `in the UltTest dir, but with no succes. What on earth am I doing wrong, here?

**Comment by PiPauwels** [comment]
OK. After some extra digging I got a new project running on web by making a small change in `package.json`:

Where, in the scripts section, it states: `"web": "ult-scripts start"`, I replaced that by `"web": "ult-scripts run-web"`. Project is running like a charm now. 

Since the scripts in `/node_module/ult-scripts/bin/ult-scripts.js` clearly lists two scripts: `run-web` and `build-web`, it seems that there is an error in the `package.json` file that is generated by default...
